relay/pkg/block/push_test.go

```diff
@@ -18,7 +18,8 @@ func TestPullHeadersFromQueue(t *testing.T) {
 	defer cancelCtx()
 
 	forwarder := &Forwarder{
-		headersQueue: make(chan *btc.Header, headersQueueSize),
+		headersQueue:              make(chan *btc.Header, headersQueueSize),
+		forwarderPullingSleepTime: defaultForwarderPullingSleepTime,
```

Comment:
Do we need to set `forwarderPullingSleepTime`? Is this test failing without that?

Reply:
It's not needed here.